rails/spec/models/story_spec.rb

```diff
@@ -26,62 +28,203 @@
     it {expect(story.language).to eq("Portuguese")}
   end
 
-  context '.import_csv' do
-    describe 'is tested against fixture file' do
-      it {expect(file_fixture('story_with_media.csv').read).not_to be_empty}
+  describe "#csv_headers" do
+    it "defines importable CSV headers" do
+      expect(described_class.csv_headers).to eq(
+        %i[
+          title
+          desc
+          permission_level
+          date_interviewed
+          language
+          interview_location_id
+          interviewer_id
+          topic
+          speakers
+          places
+          media
+        ]
+      )
```

Comment:
A question after seeing a few of these tests – does the order matter here? I'm guessing no because the `mapped_headers` hash is used to parse the CSVs right?

a small nit if it turns out order doesn't matter, I usually like to use https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/contain-exactly-matcher instead of exact array equals. I don't think this change is needed for this PR but maybe if we run into issues maintaining these tests we can introduce it.

Reply:
Ah yeah, good call! I don't believe we care about what order the headers are returned. They are extracted from the database/rails so there is some intrinsic "ordering" but that also may be different on different machines.